### PR TITLE
Package Manager Documentation

### PR DESCRIPTION
**Pull request by moxiegirl**
- Adds the library version number into the title of the reference
- Adds an explanation of CDN profile file #560, #584

Signed-off-by: Mary Anthony <mary@blockstack.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "dev-proofs": "npm run build:webpack && ts-node ./tests/browserTests/proofs-server.ts",
     "prepublishOnly": "npm run test && npm run build",
     "lint": "eslint --ext .ts ./src",
-    "build-documentation": "shx rm -rf docs && typedoc --tsconfig tsconfig.typedoc.json --out docs src && typedoc --tsconfig tsconfig.typedoc.json --json docs/docs.json src",
+    "build-documentation": "shx rm -rf docs && cross-env typedoc --name \" blockstack.js $npm_package_version Library Reference \" --tsconfig tsconfig.typedoc.json --out docs src && cross-env typedoc --name \" blockstack.js $npm_package_version Library Reference \" --tsconfig tsconfig.typedoc.json --json docs/docs.json src",
```
Comment (zone117x) [inline package.json]: Are the spaces between the name quotes intentional here?
image
image

Comment (moxiegirl) [reply package.json]: Just something i was doing during testing.

**Comment by moxiegirl** [comment]
@zone117x I fetch and incorporated your pending PR #656  before pushing this so I could test with the includes.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,25 @@
...
+  - [Releasing](#releasing)
 
 ## Installation
 
-    $ npm install blockstack
+```
+$ npm install blockstack
+```
+
+You can import `blockstack.js` as a script without using a package manager. To securely use the latest distribution of blockstack.js from a CDN, add [this script declaration](mdincludes/include:script-dist-file.md) in your application.
```
Comment (zone117x) [inline README.md]: I don't think link this works with github markdown `mdincludes/include:script-dist-file.md`

File: README.md [hunk 3]
```diff
@@ -2,21 +2,25 @@
...
--   [Auth](https://blockstack.github.io/blockstack.js/index.html#authentication)
--   [Profiles](https://blockstack.github.io/blockstack.js/index.html#profiles)
--   [Storage](https://blockstack.github.io/blockstack.js/index.html#storage)
--   [Documentation](#documentation)
--   [Compatibility](#compatibility)
--   [Contributing](#contributing)
--   [Maintainer](#maintainer)
--   [Testing](#testing)
--   [Releasing](#releasing)
+- [blockstack.js ![CircleCI](https://circleci.com/gh/blockstack/blockstack.js/tree/master) ![npm](https://www.npmjs.com/package/blockstack) ![npm](https://www.npmjs.com/package/blockstack) ![npm](https://www.npmjs.com/package/blockstack) ![Slack](http://slack.blockstack.org/)](#blockstackjs-circlecihttpscirclecicomghblockstackblockstackjstreemaster-npmhttpswwwnpmjscompackageblockstack-npmhttpswwwnpmjscompackageblockstack-npmhttpswwwnpmjscompackageblockstack-slackhttpslackblockstackorg)
```
Comment (zone117x) [inline README.md]: Markdown issue after the Slack link?

File: tsconfig.typedoc.json [hunk 4]
```diff
@@ -13,12 +13,12 @@
     "src",
   ],
   "typedocOptions": {
-    "name": "blockstack.js Library Reference",
     "readme": "overview.md",
     "mode": "file",
     "excludePrivate": true,
     "excludeProtected": true,
     "excludeExternals": true,
+    "includes" : "mdincludes"
```
Comment (zone117x) [inline tsconfig.typedoc.json]: Missing comma `"includes" : "mdincludes",`

File: README.md (hunk 2 above)
Comment (moxiegirl) [reply README.md]: I think I need  the full path there -- it won't resolve until the master merge in that case

**Review by zone117x — APPROVED** [review]
LGTM

**Pull request merged**